epub33/rs/index.html

```diff
@@ -471,8 +471,9 @@ <h3>Spine</h3>
 						<dd>
 							<p>Reading Systems MUST provide a means of rendering the EPUB Publication in the order
 								defined in the <code>spine</code>, which includes: 1) recognizing the first primary
-									<code>itemref</code> as the beginning of the default reading order; and, 2)
-								rendering successive primary items in the order given in the <code>spine</code>.</p>
+									<code>itemref</code> (i.e., whose <code>linear</code> attribute value is
+									"<code>yes</code>") as the beginning of the default reading order; and, 2) rendering
+								successive primary items in the order given in the <code>spine</code>.</p>
```

Comment:
I know that per the standard the default value of `linear` is `true` and it is not required. Nevertheless, maybe to avoid confusion, wouldn't it be better to say 

> whose <code>linear</code> attribute is missing or its value is "<code>yes</code>"

WDYT?

Reply:
The more I look at this, the more I wonder why a new term like "primary itemref" was even introduced.

The latest commit changes the term to "linear itemrefs" and links across to where the requirement for one linear item is defined in the core spec.

Reply:
That looks good to me!